receiver/pulsarreceiver/metadata.yaml

```diff
@@ -7,3 +7,6 @@ status:
   distributions: [contrib]
   codeowners:
     active: [dmitryax, dao-jun]
+tests:
+  config:
+  skip_lifecycle: true
```

Comment:
add issue to not skip these tests

Reply:
The generated code adds a TODO in the code. Good enough?

Reply:
Thanks for filing https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/29996